### ISSUE TITLE
[1.21.1] Bees won't enter Advanced Hives

### ISSUE THREAD

**Issue by Draskh** [JDKDigital/productive-bees#595]
I tried everything to make them enter their Hive, but they just fly around looking for one. 
They enter vanilla Hives regardless, though.
Also, solitary Bees won't leave their Nests for some reason unless they are forced out with a stick.

They won't even enter the Hive when it's raining or during night
![2024-11-09_06 40 32](https://github.com/user-attachments/assets/3976c49d-4087-439e-b64d-21dc5727f8ed)
[latest.log](https://github.com/user-attachments/files/17686189/latest.log)

**Comment by ahiddenwaffle** [comment]
Experiencing this issue in All The Mods 10. It seems to be a problem with only naturally spawned bees, bees spawned via spawn egg don't have issues getting into their hive.

**Comment by JaisDK** [comment]
naturally spawned bees are almost all solitary bees that don't live in hives

**Comment by ahiddenwaffle** [comment]
In my testing I fished a bee up from the ocean and he couldn't find his hive, but one spawned via egg had no issues.

**Comment by JaisDK** [comment]
did it sting you? they come out of the water angry and will sting you (and die) if you don't catch it first
it also doesn't go into a hive while angry

**Comment by ahiddenwaffle** [comment]
I fished him up, fed him a honey treat to calm him, put him in a bee cage, then shift + right click on the advanced bee hive.

**Comment by JaisDK** [comment]
and provided the flowering block?

**Comment by ahiddenwaffle** [comment]
Correct. He even pollinated them but now is stuck wandering around. Even if I trap him in a small box with just the flowering block + hive, he will not enter it.

**Comment by Vageto12** [comment]
i have this same problem on the atm10 modpack version 4.12
